packages/notebook-extension/src/index.ts

```diff
@@ -1730,7 +1730,11 @@ function addCommands(
       const current = getCurrent(args);
 
       if (current) {
-        return NotebookActions.moveUp(current.content);
+        NotebookActions.moveUp(current.content);
+        Private.raiseSilentNotification(
+          'Notebook cell shifted up successfully',
```

Comment:
Needs to be translated:

```suggestion
          trans.__('Notebook cell shifted up successfully'),
```